File: LinearSort/LinearSort1.c
#include<stdio.h>

int main()
{
int m;
m=5;
int x[m];
int y,e,f,g,oep,iep;
for(y=0;y<m;y++)
{
printf("Enter a number : ");
scanf("%d",&x[y]);
fflush(stdin);
}

oep=m-2;
iep=m-1;
e=0;
while(e<=oep)
{
f=e+1;
while(f<=iep)
{
if(x[f]<x[e])
{
g=x[e];
x[e]=x[f];
x[f]=g;
}
f++;
}
e++;
}

for(y=0;y<m;y++)
{
printf("%d\n",x[y]);
}
return 0;
}

File: LinearSort/LinearSort1.py
x=[]
req=5
for i in range(req): x.append(int(input("Enter a number : ")))
oep=req-2
iep=req-1
e=0
while e<=oep:
    f=e+1
    while f<=iep:
        if x[f]<x[e]:
            g=x[e]
            x[e]=x[f]
            x[f]=g
        f+=1
    e+=1
for i in range(req): print(x[i])
